app/validation/validator.py

```diff
@@ -128,13 +128,21 @@ def validate_questionnaire(self, json_to_validate):
 
     def _validate_section(self, section):
         errors = []
-        section_repeat = section.get("repeat", {})
+        section_repeat = section.get("repeat")
 
         if section_repeat:
             errors.extend(self._validate_list_exists(section_repeat["for_list"]))
             errors.extend(
                 self._validate_placeholder_object(section_repeat["title"], None)
             )
+
+        custom_section_summary = section.get("summary")
+        if custom_section_summary:
+            for custom_section_element in custom_section_summary.get("items", []):
```

Comment:
The naming here needs to change. The existence of a `summary` element does not mean it's a custom section summary as the `collapsible` property is also possible. A custom section summary exists if there is an `items` property.

Reply:
Changed to `section_summary` and `items`